docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_run_command/_index.md

```diff
@@ -0,0 +1,161 @@
+
+- To use PowerShell you need to check if the installed version of module _Az.ConnectedMachine_ is 0.6.0 or higher. Use the following PowerShell command to check the installed version
+
+```powershell
+Get-Module -ListAvailable -Name Az.ConnectedMachine
+```
+Use the following command to install the latest version of the PowerShell module
+
+```powershell
+Install-Module -Name Az.ConnectedMachine -Force 
+```
+
+## Use the Run command to execute a simple PowerShell command within an Arc-connected Windows machine
+
+- Run the following Azure cli command after adding the appropriate resource group, name of the Arc-connected machine, a name identifying the command and the location of you Arc-connected machine
+
+```shell
+    az connectedmachine run-command create --resource-group <Resource Group Name> --machine-name <Machine Name> --run-command-name <Identifying Name of command> --script "Write-Host 'Hello World'" --location <Location>
+```
+- It takes few minutes to run the command and return the results. If the execution is successful then you would see the following within the longer returned Json string:
+    
+```shell
+    "executionState": "Succeeded",
+    "exitCode": 0,
+    "output": "Hello World",
+```
+
+If you prefer to use PowerShell then use the following command
+
+```powershell
+New-AzConnectedMachineRunCommand -ResourceGroupName "<Resource Group Name>" -Location "<Location>" -SourceScript "Write-Host 'Hello World'" -RunCommandName "<Identifying Name of command>" -MachineName "<Machine Name>"
+```
+
+The successful output of the PowerShell command will show the following output
+
+![Screenshot success helloworld PowerShell](07.png)
+
+## Use the Run command to execute a simple Shell command within an Arc-connected Linux machine
+
+- On your client machine run the following Azure cli command after adding the appropriate parameters
+
+```shell
+    az connectedmachine run-command create --resource-group <Resource Group Name> --machine-name <Machine Name> --run-command-name <Identifying Name of command> --script "ifconfig" --location <Location>
+```
+
+Or in PowerShell
+
+```powershell
+New-AzConnectedMachineRunCommand -ResourceGroupName "<Resource Group Name>" -Location "<Location>" -SourceScript "ifconfig" -RunCommandName "<Identifying Name of command>" -MachineName "<Machine Name>"
+```
+
+- If the execution is successful then you should have an output which includes the result of the ifconfig command as a string which includes "\n" to indicate line breaks in the original output.
+    ![Screenshot](./04.png)
+
+## Examine the available options for the Run command
+
+- Run the following command which returns the available options of the Run command such as "create", "delete", "list", "show", "update" and "wait".
+
+```shell
+    az connectedmachine run-command --help
+```
+
+- Investigate the different ways you can run the "create" option by running the following command on your client machine
```

Comment:
```suggestion
- Investigate the different ways you can run the "create" option by running the following command on your client machine.
```

Reply:
Please go over the readme and use dot at the end of sentences

Reply:
done, sometimes using : instead of .